python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2550,3 +2550,27 @@ def test_parquet_reader_zstd_compression(datadir):
         assert_eq(df, pdf)
     except RuntimeError:
         pytest.mark.xfail(reason="zstd support is not enabled")
+
+
+def test_read_parquet_multiple_files(datadir):
+    df_1 = cudf.DataFrame({"id": range(100), "a": [1] * 100})
+    df_1.to_parquet(datadir / "df_1.parquet")
```

Comment:
Does this need to set the index to be the “id” column?

Reply:
I didn't fully understand your question here, could you elaborate?

Reply:
I was trying to understand if you intended `id` to be an Index for the DataFrame, like `DataFrame.set_index("id")`. Maybe not. I just wanted to ensure that you're testing what you intended to test with a "`RangeIndex` or similarly named index."

Reply:
Oh, yeah this test is written the intended way. `id` is just a random column I picked straight from the bug report. It could just be any other column.